confluent/confluent.go

```diff
@@ -57,13 +89,35 @@ func createBasicConfigMap(properties map[string]string) kafka.ConfigMap {
 	return configMap
 }
 
-// CreateTopic creates a topic using the Admin Client API.
-func CreateTopic(producer *kafka.Producer, topic string) {
+// CreateTopic creates a topic using the Admin Client API with default settings.
+// This function will panic if the topic does not exist and cannot be created
+// for any reason.
+func CreateTopic(producer k.Producer, topic string) error {
```

Comment:
do we want to support being able to create multiple topics? or is one at a time how you do it?